packages/core/copy-uuid/README.md

```diff
@@ -161,13 +161,13 @@ An indicator of whether a long UUID is truncated. When `true`, the UUID will be
 
 An indicator of whether a `.mono` class is added to the UUID string. Make sure to import the Kongponents style file in your host application for this class to take effect.
 
-### `isHidden`
+### `format`
 
-- type: `Boolean`
+- type: `String`
 - required: `false`
-- default: `false`
+- default: `uuid`
 
-An indicator of whether the UUID string is replaced with asterisks.
+Determines the display format of the UUID string.
```

Comment:
Could you list accepted values here please?

Reply:
updated.